src/raven.js

```diff
@@ -652,9 +650,9 @@ Raven.prototype = {
                 target = '<unknown>';
             }
 
-            self.captureBreadcrumb('ui_event', {
-                type: evtName,
-                target: target
+            self.captureBreadcrumb({
+                category: 'ui.' + evtName, // e.g. ui.click, ui.input
```

Comment:
i would probably give this the type "ui" in addition to category "ui.click" so that we can give it an icon.